sycl/test/string-selector/select.cpp

```diff
@@ -0,0 +1,121 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t1.out
+// RUN: %t1.out
+
+// REQUIRES: cpu, gpu, opencl
```

Comment:
The checks below are written in quite safe manner, i.e. the test should pass with even only one of CPU or GPU is available.
Can this line be: "// REQUIRES: opencl" ? or be simply removed?

Reply:
I'd prefer to check that it can do different things in the same program.